ledger/sandbox/src/main/scala/com/digitalasset/platform/apiserver/services/ApiCommandService.scala

```diff
@@ -194,7 +198,9 @@ object ApiCommandService {
       maxCommandsInFlight: Int,
       limitMaxCommandsInFlight: Boolean,
       historySize: Int,
-      retentionPeriod: FiniteDuration)
+      retentionPeriod: FiniteDuration,
+      // TODO(RC): this should be updated dynamically from the ledger configuration
```

Comment:
hrhr, `RC` ;)

Reply:
Force of habit 😄

Reply:
Apologies, I definitely didn't intend for you to change it :(
Just found it funny.